server/task/task-distributed/src/main/java/org/apache/james/task/eventsourcing/distributed/RabbitMQWorkQueue.java

```diff
@@ -117,15 +117,22 @@ private void consumeWorkqueue() {
         String json = new String(delivery.getBody(), StandardCharsets.UTF_8);
 
         TaskId taskId = TaskId.fromString(delivery.getProperties().getHeaders().get(TASK_ID).toString());
-
+        Task task;
         try {
-            Task task = taskSerializer.deserialize(json);
+            task = taskSerializer.deserialize(json);
+        } catch (Exception e) {
+            LOGGER.error("Unable to deserialize submitted Task " + taskId.asString(), e);
```

Comment:
No, because i still want to provide the source exception.
BTW i think we should revise the use of logger with placeholder and exceptions.
I have found some in the code. And if i believe the code of the logging methods the formating will fail because the number of parameters doesn't match the number of placeholder and The throwable will be set to null.
example of a badly used logging 
```
                        LOGGER.error("Unable to index message {} in mailbox {}", message.getUid(), mailbox.getName(), e);
```  in `LazyMessageSearchIndex` line 128